controllers/default.py

```diff
@@ -940,10 +940,6 @@ def mark_friend():
                                          "friend_requests",
                                          scheme=True,
                                          host=True,
-                                         extension=False),
```

Comment:
I didn't remove it... I guess git was just showing it as removal because the unsubscribe URL also has that line and it was removed for unsubscribe URL